### ISSUE TITLE
Allow resizable columns in tables

### ISSUE THREAD

**Issue by montaropdf** [RimSort/RimSort#708]
### Origin of idea

Currently, column width is fixed to the size of the window containing the table. This means that on small resolution or on small windows, the content of cells is displayed truncated and there is no way to see its full content except by increasing the resolution and the size of the window.

### Suggestion/proposed solution?

Column width should be made resizable like it it is the case with a file manager or a spreadsheet application.

### Code of Conduct

I agree to follow this project's Code of Conduct

### Duplicate Issue Check

I've checked for similar issues and didn't find anything

### Wiki/FAQ Check

I've checked the Wiki for a solution and didn't find a solution

**Comment by LionelColaso** [comment]
Can You Please explain and add screenshot of which column u have issues with

**Comment by montaropdf** [comment]
Hello,

I have issues with all tables. They all behave the same. The width of the columns are proportional to the size of the window, when they should be resizable like in applications such as a file manager or a spreadsheet.

**Comment by Artemonim** [comment]
At the moment of 1.0.32:
- the table in the ACF Log Reader section has this feature.
- the table in File Search does not have, but it rather needs a word wrap.

**Comment by oceancabbage** [comment]
https://github.com/RimSort/RimSort/pull/1173 adds resizable columns to both tables (in review)

**Comment by oceancabbage** [comment]
PR has been merged. The fix should be included the next release
